lib/sm/api/messages.rb

```diff
@@ -49,13 +49,13 @@ def post_create_message(args = {})
       end
 
       # post_create_message_draft: Creates a new message draft, or updates an existing one, without attachments.
-      def post_create_message_draft(args = {})
-        args.transform_keys! { |k| k.to_s.camelize(:lower) }
-        json = perform(:post, 'message/draft', args.to_json, token_headers)
-        json[:data].delete(:attachments)
-
-        Message.new(json)
-      end
+      # def post_create_message_draft(args = {})
+      #   args.transform_keys! { |k| k.to_s.camelize(:lower) }
+      #   json = perform(:post, 'message/draft', args.to_json, token_headers)
+      #   json[:data].delete(:attachments)
+      #
+      #   Message.new(json)
+      # end
```

Comment:
Oh, NM you moved it into a new API directory -- OK, so that's fine, but can you copy and paste the code exactly as it was including the transform keys?